Jenkinsfile

```diff
@@ -111,6 +111,10 @@ node {
             runInVagrant("cd servicetest && sudo ./run-tests.sh")
         }
 
+        stage('DocTest') {
```

Comment:
In my mind DocTest refers to a specific type of framework, e.g. https://www.reddit.com/r/cpp/comments/4kibl4/doctest_the_lightest_feature_rich_c_single_header/, which we might adopt in the future. Therefore I think a better name can be found.